src/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,35 @@
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px;
+    background-color: #d4d4d4;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    width: 100%;
+    border-radius: 10px;
+    margin-right: 10px;
+    height: 70px;
+    box-sizing: border-box;
+  }
+  
+.navList {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.navItem {
+  margin: 0 16px;
+  padding: 8px 0;
+  cursor: pointer;
+  color: #000000;
+  text-decoration: none;
+  transition: color 0.2s;
+  font-family: Roboto;
+  font-weight: 300;
+}
```

Comment:
I don't see any effect with this rules:
color, text-decoration, font-family, font-weight.
Are they needed? If not, please remove.

Reply:
For the color and text-decoration you are absolutely right. But the others are playing a role. Never the less, I didn't import the font so I fixed it. Thanks for the observation.